chart/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 
 version: 0.7.0
 appVersion: 0.7.0
-kubeVersion: ">= 1.19.0 < 1.22.0"
+kubeVersion: ">= 1.19.0 <= 1.22.0"
```

Comment:
@VineethReddy02 Is this change okay with you? I'm not sure if we had it limited to < 1.22.0 for a specific reason.

Reply:
Yup, the otel-operator doesn't support 1.22 version on uninstall flow leaves the child components. So we decided not to support 1.22 version atm.